pkg/pg/pg.go

```diff
@@ -0,0 +1,28 @@
+package pg
+
+import (
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+	"github.com/scylladb/go-reflectx"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/utils/tests"
+)
+
+func NewSqlxDB(t testing.TB, dbURL string) *sqlx.DB {
+	tests.SkipShortDB(t)
+	err := RegisterTxDb(dbURL)
+	if err != nil {
+		t.Errorf("failed to register txdb dialect: %s", err.Error())
```

Comment:
should not this be a fatal error?
Otherwise caller has to check if returned db is nil.